Runtime/Editor/StoredUser.cs

```diff
@@ -0,0 +1,111 @@
+using LootLocker;
+using LootLocker.Extension.Requests;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+#if (UNITY_EDITOR) 
+public class StoredUser : ScriptableObject
+{
+
+    
+    [HideInInspector]
+    public User user = null;
+
+    [HideInInspector]
+    public string serializedUser;
+ 
+    private void Deserialize()
+    {
+        if(user == null || user.id == 0)
+        {
+            user = LootLockerJson.DeserializeObject<User>(serializedUser);
+        }
+    }
+
+    private void Serialize()
+    {
+        if(user == null)
+        {
+            serializedUser = null;
+            return;
+        }
+        serializedUser = LootLockerJson.SerializeObject(user);
+    }
+
+    private static StoredUser Get()
+    {
+        if (_current != null)
+        {
+            return _current;
+        }
+        _current = Resources.Load<StoredUser>("/LootLockerSDK/Runtime/Editor/Resources/Config/LootLockerUser");
+
+#if UNITY_EDITOR
```

Comment:
Remove this #if UNITY_EDITOR, the whole class is wrapped inside one, so this one doesn't do anything.

Reply:
Removed :)